setup.py

```diff
@@ -50,10 +51,16 @@
         "isort",
         "flake8",
         "flake8-docstrings",
-        "pytest",
-        "pytest-cov",
+    ],
+    "examples": [
+        "jupyterlab",
+        "xarray",
+        "dask",
+        "netcdf4",
+        "cdsapi",
```

Comment:
That's a good point, I'd been starting to ponder whether they should all just be a part of the requirements, and I think this is what I needed. We also use ipython in the code now for the updated PlantData repr method, so I think it would just be easier to be more inclusive all around like we did when updating for the WTK dependency.